modules/core/arrow-core-data/src/main/kotlin/arrow/typeclasses/Apply.kt

```diff
@@ -315,4 +315,28 @@ interface Apply<F> : Functor<F> {
   ): Kind<F, Tuple10<A, B, C, D, E, FF, G, H, I, J>> =
     a.product(b).product(c).product(d).product(e).product(f).product(g)
       .product(h).product(i).product(j)
+
+  /**
+   * Given two actions, it performs them sequentially.
+   * Ignores the result of the first action.
+   *
+   * This is equivalent to *> in Haskell.
+   */
+  fun <A, B> Kind<F, A>.followedBy(fb: Kind<F, B>): Kind<F, B> =
+    map(this, fb) { (_, right) -> right }
+
+  fun <A, B> Kind<F, A>.followedByEval(fb: Eval<Kind<F, B>>): Kind<F, B> =
```

Comment:
There is sadly no point to having this on `Apply` without having a lazy `ap` method. Maybe move this back to `Monad` where it actually made sense or remove it :/

Reply:
I just move it by inertia 😅